cmd/topology/topology.go

```diff
@@ -44,8 +60,64 @@ var (
 		Short: "push or generate certs for nodes in topology",
 		RunE:  certFn,
 	}
+	resetCfgCmd = &cobra.Command{
+		Use:   "reset -skip <topology> <device>",
+		Short: "reset configuration of device (if device not provide reset all nodes)",
+		RunE:  resetCfgFn,
+	}
+)
+
+var (
+	skipReset bool
 )
 
+func resetCfgFn(cmd *cobra.Command, args []string) error {
+	if len(args) != 1 {
+		return fmt.Errorf("%s: missing args", cmd.Use)
```

Comment:
Won't this cause failure if a user specifies a specific device as well? I think you meant

if len(args) < 1